libs/blocks/global-navigation/global-navigation.css

```diff
@@ -702,5 +706,6 @@ header.global-navigation {
 
   .feds-navItem--megaMenu .feds-popup {
     align-items: center;
+    flex-wrap: wrap;
```

Comment:
Why is this needed? Is it desirable to have columns span on multiple rows in certain scenarios? Doesn't this interfere with the `feds-crossCloudMenu-wrapper` element, since it's also inside the popup? This is how this tweak affects the current Photoshop page:
<img width="1728" alt="Screenshot 2024-07-23 at 15 39 27" src="https://github.com/user-attachments/assets/465c76a4-0295-4f99-b133-b02145ddf555">

Since this doesn't seem to be related to the promo, it might be good to remove this.

Reply:
This was added to fix one of the issues where the left side of the large menu gets cut from left side (1200-1300px range). I'll remove this due to above issue and as its unrelated to promobar change. We will log a separate bug for this.

<img width="1271" alt="Screenshot 2024-07-24 at 12 47 44 PM" src="https://github.com/user-attachments/assets/ff375c30-786a-4dc2-9fb8-e0212c7b6284">

Reply:
5 columns have never been supported in the dropdown. The initial Consonant guidelines (which I can't locate currently due to how many changes they had on their platform 😅 ) stated that a maximum of 4 columns should be added in a dropdown menu.